docs/pages/configuration/dev/files/sync.mdx

```diff
@@ -507,10 +760,34 @@ By default, the file synchronization algorithm uses the maximum bandwidth possib
 
 ## Advanced Options
 
+<<<<<<< HEAD
+### `arch`
+
+Arch specifies which DevSpace helper architecture should be used for the container. Currently valid values are either no value, `amd64` or `arm64`. Depending on this value, DevSpace will inject the DevSpace helper binary with the corresponding architecture suffix.
+
+=======
+>>>>>>> upstream/v6-docs
 ### `polling`
 
 Polling specifies if the DevSpace helper should traverse over all watched files and folders periodically in the container to identify file changes. By default, DevSpace will use [inotify](https://man7.org/linux/man-pages/man7/inotify.7.html) to detect changes which is more efficient, however sometimes it might be unsupported or not feasible in certain situations, in which polling might be preferred.
```

Comment:
```suggestion
Polling specifies if the DevSpace helper should traverse over all watched files and folders periodically in the container to identify file changes. By default, DevSpace uses [inotify](https://man7.org/linux/man-pages/man7/inotify.7.html) to detect changes. This can be more efficient, however, sometimes it might be unsupported or not feasible in certain situations, in which case, polling might be preferred.
```